docs/content/s3.md

```diff
@@ -1271,13 +1271,13 @@ Properties:
 
 #### --s3-sse-customer-algorithm
 
-If using SSE-C, the server-side encryption algorithm used when storing this object in S3.
+If using SSE-C, the server-side encryption algorithm used when storing this object in S3.  As per AWS S3 standard, this must be "AES256" if SSE-C is used.
 
 Properties:
 
 - Config:      sse_customer_algorithm
 - Env Var:     RCLONE_S3_SSE_CUSTOMER_ALGORITHM
-- Provider:    AWS,Ceph,ChinaMobile,Minio
+- Provider:    AWS,Ceph,ChinaMobile,Minio, IDrive (if enabled via dashboard)
```

Comment:
This also comes from here

https://github.com/rclone/rclone/blob/87eaf376298436c7991ff6975184961bb86bc895/backend/s3/s3.go#L1970-L1972

You can have IDrive in or out - you can't have a comment! If you need a comment then put it in the [IDrive section in s3.md](https://github.com/rclone/rclone/blob/master/docs/content/s3.md#idrive-e2-idrive-e2)

The same applies to the changes below.